vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -138,7 +138,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>winsqlite3.lib;ChakraRT.lib;dxguid.lib;dloadhelper.lib;OneCoreUap_apiset.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
+      <DelayLoadDLLs>api-ms-win-core-file-l1-2-0.dll;ext-ms-win-uiacore-l1-1-1.dll;api-ms-win-core-windowserrorreporting-l1-1-0.dll;chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
>ext-ms-win-uiacore-l1-1-1.dll [](http://example.com/codeflow?start=53&length=29)

Generally speaking all ext- that you depend on must be delayloaded (and all usage of methods within needs to be gated behind an `IsAPINAMEPresent()` check).  The ext- means that they are optional and don't exist on all editions of Windows.  If RNW were to run on an edition missing this then a static dependency would cause RNW.dll to fail to load entirely.